pycbc/events/significance.py

```diff
@@ -219,6 +264,26 @@ def insert_significance_option_group(parser):
                              "Options: ["
                              + ",".join(trstats.fitalpha_dict.keys()) + "]. "
                              "Default = exponential for all")
+    parser.add_argument('--limit-ifar', nargs='+', default=[],
+                        help="Value to limit the IFAR value (years). "
+                             "Given as combination:function pairs, i.e. "
+                             "H1:10000 L1:1000. Set to 0 to allow "
+                             "unlimited IFARs (default), though "
+                             "this may cause numerical under/overflow "
+                             "errors for loud signals, e.g. injections.")
```

Comment:
if I understand correctly, this could be used to set limits for coincs as well as singles .. 
I would suggest to rephrase slightly `loud single-ifo signals or injections`

Reply:
I had the same thought, but then realised that n_louder_from_fit can _also_ be used for coincs, e.g. in the hierarchical search's significance extrapolation described in https://arxiv.org/abs/2309.00019

Reply:
See suggestion below. While editing I started to wonder what happens if someone gives values for only _some_ of the existing combos - as in your example where only the singles are given, or my new one where H1L1 and L1 are given but not H1.
My guess is that the non-specified combos are unlimited, but not sure if the code actually does that (rather than crashing)

Reply:
That is right, unspecified combos are left as unlimited.

You can also provide combinations which are not used in the code (so that the option can be reused for e.g. coinc_statmap and add_statmap) without crashing, just a warning.

Reply:
Ah, right, I didn't see the `_default_opt_dict`.